package.json

```diff
@@ -11,7 +11,7 @@
     "prepare": "npm run build",
     "build": "tsc --project ./tsconfig.json",
     "test": "BLUEBIRD_DEBUG=1 node --max_old_space_size=3072 node_modules/jasmine/bin/jasmine.js --stop-on-failure=true",
-    "lint": "eslint -c .eslintrc --max-warnings 0 src/**/*.ts",
```

Comment:
I totally misunderstood you in bridge crew. We want to keep this for now, as we're trying to keep the warnings at 0.